ci/check.sh

```diff
@@ -37,12 +37,11 @@ function run() {
 
 parent=$(cd $(dirname $0) && pwd -P)
 
-banner "Executing in conda environment ${CONDA_DEFAULT_ENV} in directory fgpyo"
-run "Style Checking" "ruff format fgpyo"
-run "Linting"        "ruff check --fix fgpyo"
-run "Type Checking"  "mypy -p fgpyo --config $parent/mypy.ini"
+run "Style Checking" "ruff format fgpyo tests"
+run "Linting"        "ruff check --fix fgpyo tests"
+run "Type Checking"  "mypy fgpyo tests --config $parent/mypy.ini"
 run "Unit Tests"     "python -m pytest -vv -r sx tests"
-run "Make docs"      "poetry run mkdocs build --strict"
+run "Make docs"      "mkdocs build --strict"
```

Comment:
I have aliased `poetry` to an installation in a dedicated environment, per [Poetry's official recommendation/requirement](https://python-poetry.org/docs/#installing-with-pipx), and `run` does not expand the alias. 

>Poetry should always be installed in a dedicated virtual environment to isolate it from the rest of your system. It should in no case be installed in the environment of the project that is to be managed by Poetry. This ensures that Poetry’s own dependencies will not be accidentally upgraded or uninstalled. (Each of the following installation methods ensures that Poetry is installed into an isolated environment.) In addition, the isolated virtual environment in which poetry is installed should not be activated for running poetry commands.

i.e. `alias poetry=/Users/msto/.miniforge-x86/envs/poetry/bin/poetry`

The other lints are not run via poetry, so this is also more consistent with the other lints.

Reply:
TIL